### ISSUE TITLE
Xposed v89 SDK26 crash on module download page

### ISSUE THREAD

**Issue by codeHusky** [rovo89/XposedInstaller#420]
When opening a module's download page, the app immediately crashes.

v89 arm64 beta 1 sdk26 G8441 (Xperia XZ1 Compact)

**Comment by codeHusky** [comment]
Additionally it seems opening a module's download page from the browser first rather than the browse page built in prevents the crash from occurring

**Comment by codeHusky** [comment]
What??

**Comment by rovo89** [comment]
I assume you're referring to the original Xposed Installer app, version 3.1.4?  I can't reproduce it here, opening the download page of any module and downloading it works without problems. Could you please post a logcat of the crash and provide exact steps to reproduce?

The installed framework (v90-beta1, btw) shouldn't have any impact on that.

**Comment by roblee78** [comment]
God I need a new pair of shoes these are worn out and stink? And am I going
to be a construction worker when i grow up and teach how to be an
electrician so I can make a space ship.

**Comment by wanam** [comment]
Here is the crash, i didn't notice it because i rarely download modules within the app.
I guess it's not related to the framework, you can reproduce it by opening the download page, search a module, and click on one of the results:
```
1-09 08:07:56.487 23442 23442 E AndroidRuntime: FATAL EXCEPTION: main
01-09 08:07:56.487 23442 23442 E AndroidRuntime: Process: de.robv.android.xposed.installer, PID: 23442
01-09 08:07:56.487 23442 23442 E AndroidRuntime: java.lang.IllegalStateException: Handling non empty state of parent class is not implemented
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at se.emilsjolander.stickylistheaders.StickyListHeadersListView.onSaveInstanceState(StickyListHeadersListView.java:1098)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.View.dispatchSaveInstanceState(View.java:17614)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:3720)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:3726)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:3726)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.ViewGroup.dispatchSaveInstanceState(ViewGroup.java:3726)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.view.View.saveHierarchyState(View.java:17597)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.FragmentManagerImpl.saveFragmentViewState(FragmentManager.java:2639)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.FragmentManagerImpl.saveFragmentBasicState(FragmentManager.java:2660)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.FragmentManagerImpl.saveAllState(FragmentManager.java:2713)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.FragmentController.saveAllState(FragmentController.java:112)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.Activity.onSaveInstanceState(Activity.java:1569)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.support.v4.app.FragmentActivity.onSaveInstanceState(FragmentActivity.java:570)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.support.v7.app.AppCompatActivity.onSaveInstanceState(AppCompatActivity.java:509)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at de.robv.android.xposed.installer.WelcomeActivity.onSaveInstanceState(WelcomeActivity.java:209)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.Activity.performSaveInstanceState(Activity.java:1496)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnSaveInstanceState(Instrumentation.java:1386)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread.callCallActivityOnSaveInstanceState(ActivityThread.java:4721)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread.performStopActivityInner(ActivityThread.java:4025)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread.handleStopActivity(ActivityThread.java:4084)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread.-wrap24(Unknown Source:0)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1622)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6494)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
01-09 08:07:56.487 23442 23442 E AndroidRuntime: 	at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:108)
01-09 08:07:56.489  1184  1886 W ActivityManager:   Force finishing activity de.robv.android.xposed.installer/.DownloadDetailsActivity
01-09 08:07:56.492  1184  1549 I ActivityManager: Showing crash dialog for package de.robv.android.xposed.installer u0
```
It's [already reported](https://github.com/emilsjolander/StickyListHeaders/issues/477) a bug on StickyListHeaders lib.

**Comment by wanam** [comment]
@brianguertin [said](https://github.com/emilsjolander/StickyListHeaders/issues/477#issuecomment-345310364) it's related to the auto-fill framework, as a workaround we can still disable this feature on the list:

```java
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
  myList.setImportantForAutofill(IMPORTANT_FOR_AUTOFILL_NO_EXCLUDE_DESCENDANTS);
}
```

An other fix on [a fork](https://github.com/mtotschnig/StickyListHeaders/commit/c9f9a44cb689320fa2e22496460a83264d1b3020).

**Comment by rovo89** [comment]
Thanks! The key was searching for a module - just scrolling through the list works fine for me.

**Comment by codeHusky** [comment]
Noticed that last night, forgot to mention it. Oopsies.

**Comment by cbartondock** [comment]
Can confirm the issue. My first thought was that it might be a webview thing.

**Comment by rovo89** [comment]
Fixed in 3.1.5.
